src/class-wpml-elementor-data-settings.php

```diff
@@ -124,7 +124,12 @@ public function get_fields_to_save() {
 	 * @return array
 	 */
 	public function add_data_custom_field_to_md5( array $custom_fields_values, $post_id ) {
-		$custom_fields_values[] = get_post_meta( $post_id, $this->get_meta_field(), true );
+		if ( apply_filters( 'wpml_pb_is_auto_update_activated', false ) ) {
```

Comment:
@strategio as asked in the other PR, can't you just check for `defined( 'WPML_TM_VERSION' )`?